### PR TITLE
Bscharenberg/wqxr on fastboot

### PR DESCRIPTION
**Pull request by bryn19**
update to account for changes in requesting flatpages and django pages

### CODE REVIEW

File: app/templates/flat-page.hbs [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        <div class="graphic-default">
+          <div class="grid_8 main l-col2of3">
+            <div class="page-titlegroup">
+              <h1 class="h1 h1--extrabold h1--mq-small">{{flatPage.title}}</h1>
+            </div>
+            <div class="article-tools clearfix">
+              <div class="share-tools clearfix">
+                <ul class="sharelinks clearfix">
+                  <li class="fa-hover facebook">
+                    <a title="Share on Facebook" class="fa-hover-link" href="https://www.facebook.com/sharer/sharer.php?u=http%3A%2F%2Fwww.wqxr.org%2Fabout%2F" data-share-via="facebook" data-story-title="About+WQXR" data-show-name="About+WQXR">
```
Comment (kedmundson) [inline app/templates/flat-page.hbs]: Looks like there is some static content left over from a possible view source copy / paste in this section. (See share href and data-attrs.)

File: app/templates/flat-page.hbs [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                    <a title="Share on Twitter" class="fa-hover-link" href="https://twitter.com/intent/tweet/?text=About WQXR&amp;via=WQXR&amp;url=http%3A%2F%2Fwww.wqxr.org%2Fabout%2F" data-share-via="twitter" data-story-title="About+WQXR" data-show-name="About+WQXR">
+                      <i class="fa fa-twitter fa-lg"></i>
+                    </a>
+                  </li>
+                  <li class="fa-hover emails">
+                    <a title="Email a friend" class="fa-hover-link" href="mailto:?subject=About+WQXR&amp;body=http%3A%2F%2Fwww.wqxr.org%2Fabout%2F" data-share-via="email" data-story-title="About+WQXR" data-show-name="About+WQXR">
+                      <i class="fa fa-envelope fa-lg"></i>
+                    </a>
+                  </li>
+                  <li class="stfh edit-item" style="display: list-item;"><a data-url="flatpages/flatpage/83" class="stf " target="_blank" href="https://internal.wnyc.org/admin/flatpages/flatpage/83">Edit This</a></li>
```
Comment (kedmundson) [inline app/templates/flat-page.hbs]: static edit link

File: app/templates/flat-page.hbs [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+            <div class="article-tools clearfix">
+              <div class="share-tools clearfix">
+                <ul class="sharelinks clearfix">
+                  <li class="fa-hover facebook">
+                    <a title="Share on Facebook" class="fa-hover-link" href="https://www.facebook.com/sharer/sharer.php?u=http%3A%2F%2Fwww.wqxr.org%2Fabout%2F" data-share-via="facebook" data-story-title="About+WQXR" data-show-name="About+WQXR">
+                      <i class="fa fa-facebook fa-lg"></i>
+                    </a>
+                  </li>
+                  <li class="fa-hover twitter">
+                    <a title="Share on Twitter" class="fa-hover-link" href="https://twitter.com/intent/tweet/?text=About WQXR&amp;via=WQXR&amp;url=http%3A%2F%2Fwww.wqxr.org%2Fabout%2F" data-share-via="twitter" data-story-title="About+WQXR" data-show-name="About+WQXR">
```
Comment (kedmundson) [inline app/templates/flat-page.hbs]: more static share stuff

File: app/templates/flat-page.hbs [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+                      <i class="fa fa-twitter fa-lg"></i>
+                    </a>
+                  </li>
+                  <li class="fa-hover emails">
+                    <a title="Email a friend" class="fa-hover-link" href="mailto:?subject=About+WQXR&amp;body=http%3A%2F%2Fwww.wqxr.org%2Fabout%2F" data-share-via="email" data-story-title="About+WQXR" data-show-name="About+WQXR">
+                      <i class="fa fa-envelope fa-lg"></i>
+                    </a>
+                  </li>
```
Comment (kedmundson) [inline app/templates/flat-page.hbs]: more static share stuff

File: app/templates/flat-page.hbs [hunk 5]
```diff
@@ -20,21 +20,21 @@
               <div class="share-tools clearfix">
                 <ul class="sharelinks clearfix">
                   <li class="fa-hover facebook">
-                    <a title="Share on Facebook" class="fa-hover-link" href="https://www.facebook.com/sharer/sharer.php?u=http%3A%2F%2Fwww.wqxr.org%2Fabout%2F" data-share-via="facebook" data-story-title="About+WQXR" data-show-name="About+WQXR">
+                    <a class="fa-hover-link" href="https://www.facebook.com/sharer/sharer.php?u={{url-encode absoluteURL}}" type="Facebook" data-action="Share Flatpage" data-label={{concat flatPage.title ' | Facebook'}}>
```
Comment (kedmundson) [inline app/templates/flat-page.hbs]: This is a kinda nontraditional way to use the `type` attribute, it's usually for the media type of the linked document. What's your intent here?

File: app/templates/flat-page.hbs [hunk 6]
```diff
@@ -20,21 +20,21 @@
...
                       <i class="fa fa-facebook fa-lg"></i>
                     </a>
                   </li>
                   <li class="fa-hover twitter">
-                    <a title="Share on Twitter" class="fa-hover-link" href="https://twitter.com/intent/tweet/?text=About WQXR&amp;via=WQXR&amp;url=http%3A%2F%2Fwww.wqxr.org%2Fabout%2F" data-share-via="twitter" data-story-title="About+WQXR" data-show-name="About+WQXR">
+                    <a class="fa-hover-link" href="https://twitter.com/intent/tweet?url={{url-encode absoluteURL}}&text={{url-encode flatPage.title}}&via=WQXR" type="Twitter" data-action="Share Flatpage" data-label={{concat flatPage.title ' | Twitter'}}>
```
Comment (kedmundson) [inline app/templates/flat-page.hbs]: same with `type` attr

File: app/templates/flat-page.hbs [hunk 7]
```diff
@@ -20,21 +20,21 @@
...
                       <i class="fa fa-twitter fa-lg"></i>
                     </a>
                   </li>
                   <li class="fa-hover emails">
-                    <a title="Email a friend" class="fa-hover-link" href="mailto:?subject=About+WQXR&amp;body=http%3A%2F%2Fwww.wqxr.org%2Fabout%2F" data-share-via="email" data-story-title="About+WQXR" data-show-name="About+WQXR">
+                    <a class="fa-hover-link" href="mailto:?subject={{url-encode flatPage.title}}&body={{url-encode absoluteURL}}" type="Email" data-actin="Share Flatpage" data-label={{concat flatPage.title ' | Email'}}>
```
Comment (kedmundson) [inline app/templates/flat-page.hbs]: same with `type` attr. also spotted a typo: `data-actin`

File: app/templates/flat-page.hbs (hunk 5 above)
Comment (bryn19) [reply app/templates/flat-page.hbs]: ahh yes that was a mistake. I was matching what they did for the story template share buttons, but didn't realize type was just a variable and not an attribute. I'll change that.

Comment (bryn19) [reply app/templates/flat-page.hbs]: what would be a better tag for that sort of info? just go back to title?

Comment (kedmundson) [reply app/templates/flat-page.hbs]: What's the intention, describing the link for accessibility? If so, `aria-label` is best if the link doesn't actually wrap text, like in your case.

Comment (kedmundson) [reply app/templates/flat-page.hbs]: https://a11yproject.com/posts/title-attributes/

Comment (bryn19) [reply app/templates/flat-page.hbs]: yes, that was the intention. I'll make it an aria-label

File: app/templates/flat-page.hbs [hunk 8]
```diff
@@ -20,17 +20,17 @@
               <div class="share-tools clearfix">
                 <ul class="sharelinks clearfix">
                   <li class="fa-hover facebook">
-                    <a class="fa-hover-link" href="https://www.facebook.com/sharer/sharer.php?u={{url-encode absoluteURL}}" type="Facebook" data-action="Share Flatpage" data-label={{concat flatPage.title ' | Facebook'}}>
+                    <a class="fa-hover-link" href="https://www.facebook.com/sharer/sharer.php?u={{url-encode absoluteURL}}" aria-label="Facebook" data-action="Share Flatpage" data-label={{concat flatPage.title ' | Facebook'}}>
```
Comment (kedmundson) [inline app/templates/flat-page.hbs]: One more thing though. You want the text alternative to be a useful description of the action, so "Share on Facebook" is more useful that just "Facebook". same applies to the other two share actions. Here's the a11y slide deck for some good review: https://docs.google.com/presentation/d/1qCg2VjLrr6iuD6qxTU_8rYLA5vaR_OY6WHjCf9P00Xk/edit#slide=id.g52a3691329_0_0

**Pull request merged**